requirements/run.txt

```diff
@@ -1,5 +0,0 @@
-numpy
```

Comment:
we probably need some of these, is there a reason these are getting deleted?

Reply:
My apologies. It was my oversight.